src/content/use-the-api/kubectl-gs/template-cluster.md

```diff
@@ -658,6 +704,112 @@ status:
   ready: false
 ```
 
+{{< /tab >}}
+{{< tab id="command-output-azure-capi" for-impl="capz_vms">}}
+
+```yaml
+---
+apiVersion: v1
+data:
+  values: |
+    connectivity:
+      bastion:
+        enabled: true
+    controlPlane:
+      replicas: 3
+    metadata:
+      description: Development Cluster
+      name: dev01
+      organization: acme
+    providerSpecific:
+      location: germanywestcentral
+      subscriptionId: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  labels:
+    giantswarm.io/cluster: dev01
+  name: dev01-userconfig
+  namespace: org-acme
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: dev01
+  namespace: org-acme
+spec:
+  catalog: cluster
+  config:
+    configMap:
+      name: ""
+      namespace: ""
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
+  name: cluster-azure
+  namespace: org-acme
+  userConfig:
+    configMap:
+      name: dev01-userconfig
+      namespace: org-acme
+  version: 0.0.29
+---
+apiVersion: v1
+data:
+  values: |
+    clusterName: dev01
+    organization: acme
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  labels:
+    giantswarm.io/cluster: dev01
+  name: dev01-default-apps-userconfig
+  namespace: org-acme
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/cluster: dev01
+    giantswarm.io/managed-by: cluster
+  name: dev01-default-apps
+  namespace: org-acme
+spec:
+  catalog: cluster
+  config:
+    configMap:
+      name: dev01-cluster-values
+      namespace: org-acme
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
```

Comment:
```suggestion
  kubeConfig:
    inCluster: true
```
simplifying

Reply:
We still need `kubeConfig.inCluster=true`